saviSimulation/savi/commandStation/CommandStationSocketConnector.java

```diff
@@ -0,0 +1,38 @@
+package savi.commandStation;
```

Comment:
This class needs some documentation/ code comments. At least a few lines for the method prototypes would be helpful.